src/components/MessageInput/MessageInput.js

```diff
@@ -873,7 +875,7 @@ class MessageInput extends PureComponent {
           */}
 
         <ActionSheetAttachment
-          setAttachActionSheetRef={this.setInputBoxRef}
+          setInputBoxRef={this.setAttachActionSheetRef}
```

Comment:
```suggestion
          setAttachActionSheetRef={this.setAttachActionSheetRef}
```

Reply:
Also you will need to change the prop in ActionSheetAttachment component from `setInputBoxRef` to `setAttachActionSheetRef`

Reply:
Also make sure to change the typescript declaration file as well :)